_posts/2020-08-18-wtf-is-white-culture.md

```diff
@@ -0,0 +1,90 @@
+mirror version that it has inflicted on those included.
+
+In particular within the US (which is the only place I've lived and really
+experienced whiteness, so the only place I'm really qualified to address), I
+think the twin projects of the eradication and oppression of the indigenous
+people, and the kidnapping and enslavement of Black people were important
+factors that are pretty specific to the US context. And I think the erasure of
+the cultures from which whiteness was made was key in white people justifying
+these two atrocities to themselves.
+
+One example of how this is mirrored as violence against those excluded from
+whiteness is the way that many Black people in the US were severed from their
+historical cultures via the slave trade. These people lost their names and
+languages and were forced with literal violence to abandon the cultures of their
+peoples. There are, of course, numerous other examples in addition.
+
+This particular case led me to reflect on the fact that when white people say
+things like, "I am 50% French, 25% German and 25% English," Black people will
+often sort of eye roll and say, "The white people are at it again." Because this
+is a particular thing that many of them can never really know about themselves.
+Certainly not simply by asking their grandparents about it or whatever. The
+knowledge is lost—was taken from them by white people.
+
+But what drives us to discuss this pedigree sort of information amongst
+ourselves? Why does this constitute an interesting conversation topic? I think
+it may be an unconscious reaction to the fact that whiteness isn't, actually, a
+culture. The fact that people can and do talk about these percentages of
+parentage generally indicates a few generations of separation from the named
+cultures. And I think those of us who are so removed from that list… the only
+connection we really have to those cultures is the names and those percentages.
+
+What does it mean that I'm half Irish and half German? I have no idea. I don't
+really know what it means to be Irish or German. I was not really raised in a
+household where those were very meaningful and I think it is very typical for
+white kids' families (at least of my age) for it to have even less of an impact
+on their home experience than mine.
+
+In order for white people to cease the damage we do and undo the damage we have
+done to other people's cultures (and to people sort of… _through_ their
+culture or with respect to their culture) we will have to rediscover for
+ourselves what it means to have a culture.
+
+I suspect that this does not mean looking backwards. This doesn't mean
+reconnecting with your (perceived) Nordic heritage as an excuse for racism and
+ideas of Nordic racial purity (there are _no Nazis in Valhalla_, damnit), but it
+also doesn't mean putting your Scottish heritage on a pedestal by going to a
+weekend festival from time to time with bagpipes and caber tossing. We're so
```

Comment:
Not disagreeing with you here, but something I’ve been thinking about a fair bit over this past year is how disconnected white people in America really are from their homeland. I don’t think this is something which can quite be written off, because, like, we all speak English. English is a language which developed over thousands of years in a very particular locality, which many of us have never set foot upon. Many of us have cultures and traditions which harken back to that locality, or others like it. I think about literally, like, cryptids, and how our language is full of goblins and sprites and fae folk, all of whom are denizens of an entirely different continent from where we all live. American cryptids simply don’t have that same kind of history, because ours isn’t an American tongue.

So I think there *are* lines of solidarity which might be possible to be had with diasporic and immigrant communities, who similarly try to maintain a sameness of culture despite being displaced from their homeland. This, of course, is predicated on first divesting our cultures from whiteness, however.